### PR TITLE
Fade effect on uploaded background img

### PR DESCRIPTION
**Pull request by dlipeles**
Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave-intl/publishers/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Tagged reviewers.

Test Plan:

Reviewer Checklist:

Tests
- [ ] Adequate test coverage exists to prevent regressions

Security:
- [ ] No raw SQL -- Always prefer ActiveRecord query helpers ([more info on StackOverflow](https://stackoverflow.com/questions/41410752/rails-5-sql-injection#41452695))
- [ ] XSS is mitigated -- Avoid `html_safe` and `raw`; escape untrusted content from users and 3rd party APIs ([Rails XSS guide](https://brakemanpro.com/2017/09/08/cross-site-scripting-in-rails) and [OWASP XSS Prevention Cheat Sheet](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet))

### CODE REVIEW

File: app/javascript/packs/brave_rewards_banner_display.jsx [hunk 1]
```diff
@@ -68,6 +68,21 @@ class BraveRewardsPageForm extends React.Component {
     }
   }
 
+  backgroundFade() {
+    if(this.props.editMode){
```
Comment (yachtcaptain23) [inline app/javascript/packs/brave_rewards_banner_display.jsx]: Looks good, would you fix the indentation?

Comment (dlipeles) [reply app/javascript/packs/brave_rewards_banner_display.jsx]: Fixed

File: app/javascript/packs/brave_rewards_banner_display.jsx [hunk 2]
```diff
@@ -68,6 +68,21 @@ class BraveRewardsPageForm extends React.Component {
...
+      let bgClass = document.getElementsByClassName('sc-EHOje')[0].classList[1];
+      let bgElement = document.getElementsByClassName('sc-EHOje ' + bgClass)[0];
+      let bgStyle = getComputedStyle(bgElement);
+      let bgUrl = bgStyle.background.match(/url\(([^)]+)\)/i)[1]
+      let bgFaded = 'linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(' + bgUrl + ')';
+
+    //Check for default image
+    if(!bgUrl.includes('/rewards/siteBanner/assets/')){
```
Comment (yachtcaptain23) [inline app/javascript/packs/brave_rewards_banner_display.jsx]: Would you fix indentation here?

File: app/javascript/packs/brave_rewards_banner_display.jsx [hunk 3]
```diff
@@ -68,6 +68,21 @@ class BraveRewardsPageForm extends React.Component {
     }
   }
 
+  backgroundFade() {
+    if(this.props.editMode){
+      let bgClass = document.getElementsByClassName('sc-EHOje')[0].classList[1];
+      let bgElement = document.getElementsByClassName('sc-EHOje ' + bgClass)[0];
+      let bgStyle = getComputedStyle(bgElement);
+      let bgUrl = bgStyle.background.match(/url\(([^)]+)\)/i)[1]
```
Comment (yachtcaptain23) [inline app/javascript/packs/brave_rewards_banner_display.jsx]: missing semicolon

File: app/javascript/packs/brave_rewards_banner_display.jsx [hunk 4]
```diff
@@ -86,6 +101,7 @@ class BraveRewardsPageForm extends React.Component {
 
   handleBackgroundImageChange(event) {
     this.setState({backgroundImage: URL.createObjectURL(event.target.files[0])});
+    this.backgroundFade();
     this.attemptToApplyFade();
```
Comment (yachtcaptain23) [inline app/javascript/packs/brave_rewards_banner_display.jsx]: Would you delete the old code?

File: app/javascript/packs/brave_rewards_banner_display.jsx (hunk 2 above)
Comment (dlipeles) [reply app/javascript/packs/brave_rewards_banner_display.jsx]: Fixed

File: app/javascript/packs/brave_rewards_banner_display.jsx (hunk 3 above)
Comment (dlipeles) [reply app/javascript/packs/brave_rewards_banner_display.jsx]: Fixed

File: app/javascript/packs/brave_rewards_banner_display.jsx (hunk 4 above)
Comment (dlipeles) [reply app/javascript/packs/brave_rewards_banner_display.jsx]: Done

**Pull request merged**